en/middleware/drivers.md

```diff
@@ -1,8 +1,9 @@
 # Driver Development
 
-The PX4 codebase uses a lightweight, unified driver abstraction layer: [DriverFramework](https://github.com/px4/DriverFramework). New drivers for POSIX and [QuRT](https://en.wikipedia.org/wiki/Qualcomm_Hexagon) are written against this framework.
+NuttX device drivers are based on the [Device](https://github.com/PX4/Firmware/tree/master/src/drivers/device) framework.
```

Comment:
@dagar Are other platforms using DriverFramework or something else? If they are then I should add a note to make that clear.

Reply:
NuttX doesn't use DriverFramework at all. Linux and QuRT were almost exclusively using DriverFramework until recently.

Reply:
Thanks. What is Linux using now? You mentioned porting NuttX drivers, but I didn't get the impression that had happened yet.

Reply:
Still DriverFramework for the core pieces. A handful of "nuttx" i2c drivers (mostly airspeed sensors) now work on linux, but the rest can be ported fairly easily. The spi device driver layer for linux needs a little bit of work, but then the nuttx spi drivers can be ported as well. At that point nearly all of DriverFramework will be redundant.

Reply:
Thank you. I have modified this section to make the current status clear: https://github.com/PX4/Devguide/pull/390/commits/5a4330ce2db6bf57b42ec3ecdc9b563a336621c1

I've indicated that the migration will happen "over coming releases".